x-pack/plugins/siem/server/lib/uncommon_processes/elasticsearch_adapter.ts

```diff
@@ -75,7 +75,7 @@ export const getHits = (
 export const getHosts = (buckets: ReadonlyArray<{ key: string; host: HostHits }>) =>
   buckets.map(bucket => ({
     id: [bucket.key],
-    name: bucket.host.hits.hits[0]._source.host.name,
+    name: get('host.name', bucket.host.hits.hits[0]._source),
```

Comment:
Nice solution! But I would be more confident if put it like this, not sure if buckets always return with elements or elements in expected format.
```
get(`host.hits.hits.0._source.host.name`, bucket),
```

Reply:
sure thing

Reply:
i updated it, lmk what you think